kono-api/src/api/v1/image/image.control.js

```diff
@@ -81,4 +88,49 @@ export const count = async (req, res) => {
         res.send({ msg: 'server error' });
     }
 
-}
\ No newline at end of file
+}
+
+/* For post upload. */
+
+class MimetypeError extends Error {
+    constructor(...params) {
+        super(...params);
+        this.message = 'Image file mimetype undefined or unsupported.';
+    }
+}
+
+const storage = multer.diskStorage({
+    destination: (req, file, cb) => {
+        cb(null, process.env.UPLOAD_DIR)
+    },
+    filename: (req, file, cb) => {
+        if (!UPLOAD_MIME_TYPES.includes(file.mimetype)) {
+            return cb(new MimetypeError(), '');
+        }
+        const extension = file.mimetype.split('/').pop();
+        cb(null, `${uniqueString()}.${extension}`);
+    }
+});
+
+const uploadFiles = multer({ 
+    storage: storage,
+    limits: {
+        fileSize: UPLOAD_MAX_FILE_SIZE
+    }
+}).array(UPLOAD_KEY, UPLOAD_MAX_NUM_FILES);
```

Comment:
`multer` requires specific field for the request. Manually setting it as an api promise might not be a good idea so feel free to suggest if you have a better idea.